Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2012,7 +2012,28 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
      */
     private func register(_ bundle: DocumentationBundle) throws {
         try shouldContinueRegistration()
-        
+
+        let currentFeatureFlags: FeatureFlags?
+        if let bundleFlags = bundle.info.featureFlags {
+            currentFeatureFlags = FeatureFlags.current
+            FeatureFlags.current.loadFlagsFromBundle(bundleFlags)
+
+            for unknownFeatureFlag in bundleFlags.unknownFeatureFlags {
```

Comment:
Possible enhancement:
If we have the string values for all currently supported feature flags somewhere we could use `NearMiss/bestMatches(for:against:)` here to detect minor typos and make suggestions to replace the unknown flag with the closest spelled supported flag.

Reply:
I added a blurb with suggestions via `NearMiss` in [3714b75](https://github.com/apple/swift-docc/pull/891/commits/3714b7537ac8e9cd8d813582f57be06c3cc8f8c3).